docs/gallery/interactive/pyfar_orientations.ipynb

```diff
@@ -0,0 +1,133 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Orientations\n",
+    "\n",
+    "The `Orientations()` class is designed for storing, manipulating, and accessing orientation vectors. Examples for this are orientations of directional loudspeakers during measurements or head orientations. It is good to know that `Orientations` is inherited from [scipy.spatial.transform.Rotation](https://pyfar.readthedocs.io/en/latest/classes/pyfar.coordinates.html)`scipy.spatial.transform.Rotation` and that all methods of this class can also be used with `Orientations`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyfar as pf\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Entering orientations\n",
+    "\n",
+    "Lets go ahead and create an object and show the result"
```

Comment:
Maybe also mention in the text that orientations are defined by a view and up vector in cartesian coordinates and that wnew and up must be orthogonal to each other.